README.md

```diff
@@ -99,3 +120,88 @@ to have it available in your module, and make sure you use the correct virtual e
 ### Running faunanet 
 faunanet provides its own, small, REPL for interacting with a running instance. This can be used to start, stop, pause or continue it, to change classifier models or to query it's current state, input and output folders and so on. To get an overview over the available commands, you can just type  ```faunanet``` in a terminal with the virtual environment being activated that faunanet has been installed into. Alternatively, refer to the documentation.
 
+
+### Using the docker image
+
+#### Run the docker-hub image
+You can also run `faunanet` in docker by pulling the latest `faunanet` image from docker-hub and 
+running it via terminal command: 
+```bash
+docker run -ti --rm \ 
+     -v /path/on/host/for/faunanet/configs:/root/faunanet_config \
+     -v /path/on/host/for/faunanet/output:/root/faunanet_output \ 
+     -v /path/on/host/for/faunanet/models:/root/faunanet/models \ 
```

Comment:
generally, some of the `faunanet` folders are named with an underscore, some are subfolders. So finally, you have `~/.config/faunanet` and then `faunanet/models`, `faunanet_output` and `faunanet_data`. Perhaps `output` and `data` could be subfolders of `faunanet`.

Also, I am currently getting permission errors with cleaning up the folders and creating new ones, not sure what happened here, but am looking into it.

Reply:
a few issues, maybe not all so important, but just to note - did not want to open "real" issues for these, rather are review comments:
- all files generated by the docker container have different permissions and then need sudo privileges to be removed
- when building the docker container yourself, you have to be in the `docker` subdirectory because of the explicit reference of the entrypoint `"./startup_docker.py"` - this could also be a script installed alongside `faunanet`
- the optional dependencies `tensorflow-flite` or `tensorflow` are referred to in an inconsistent manner, would name this option always the same (for pip install, dockerhub option, docker custom build from dockerfile, are there any others?)
- Make sure that user knows they have to select one of these options, otherwise they cannot run `faunanet`. So `tensorflow` is not really an optional dependency, maybe set a default for one of the two options if users fail to select one option explicitly
-

Reply:
- [] permission issue: This [seems to be quite involved](https://medium.com/@nielssj/docker-volumes-and-file-system-permissions-772c1aee23ca) and difficult to do in a generic way? maybe it's worth making an extra issue for that
- [] directory structure: changing that across the entire project probably would make sense to have an extra issue too, because the folders are refered to in the tests.
- [x] startup script: I think I managed to make it independent of the dir you are in by installing all the files with the package, though it's a little hacky in the container.
- [x] optional dependencies: added remark in readme, and added default option to docker file. Fixed inconsistent readme entry.  left the docker image names as they are, however, I believe the mapping tf -> tensorflow and tflite-> tensorflow-lite is easy enough?